bokeh/sphinxext/bokeh_model.py

```diff
@@ -86,14 +87,14 @@ def run(self):
         if type(model) != Viewable:
             pass
 
-        # model_obj = model()
+        model_obj = model()
 
-        model_json = "" #json.dumps(
-        #     json.loads(serialize_json(model_obj.dump(changed_only=False, validate=False))),
-        #     sort_keys=True,
-        #     indent=2,
-        #     separators=(',', ': ')
-        # )
+        model_json = json.dumps(
+            json.loads(serialize_json(model_obj.vm_serialize(False))),
```

Comment:
a cleaner way might be to add `PlotObject.to_json_string` which would return `serialize_json(vm_serialize())`, and `PlotObject.to_json` which would return `loads(to_json_string)`. That would keep _json_encoder from leaking, I think it's pretty much an implementation detail - nobody outside of the bokeh module wants "JSON with weird stuff in it that only our custom encoder can handle"